lambdas/convert-set-to-old-format/src/test/java/uk/gov/di/data/lep/ConvertSetToOldFormatTest.java

```diff
@@ -1,11 +1,23 @@
+package uk.gov.di.data.lep;
```

Comment:
Oh, I moved all these into a package, not sure why they weren't before but this seemed to make sense